applications/CompressiblePotentialFlowApplication/python_scripts/define_wake_process_2d.py

```diff
@@ -62,16 +45,36 @@ def __init__(self, Model, settings):
                 node.Set(KratosMultiphysics.SOLID)
 
     def ExecuteInitialize(self):
+        self.__FindWakeElements()
+
+    def __FindWakeElements(self):
+        self.__ReadWakeDirection()
         # Save the trailing edge for further computations
-        self.SaveTrailingEdgeNode()
+        self.__SaveTrailingEdgeNode()
         # Check which elements are cut and mark them as wake
-        self.MarkWakeElements()
+        self.__MarkWakeElements()
         # Mark the elements touching the trailing edge from below as kutta
-        self.MarkKuttaElements()
+        self.__MarkKuttaElements()
         # Mark the trailing edge element that is further downstream as wake
-        self.MarkWakeTEElement()
+        self.__MarkWakeTEElement()
+
+    def __ReadWakeDirection(self):
```

Comment:
```suggestion
    def _SetWakeDirectionAndNormal(self):
```
You are not reading it.

Reply:
Ok I agree!